mod/register.php

```diff
@@ -134,7 +134,7 @@ function register_post(App $a)
 		}
 
 		// send email to admins
-		$admin_mail_list = "'" . implode("','", array_map(dbesc, explode(",", str_replace(" ", "", $a->config['admin_email'])))) . "'";
+		$admin_mail_list = "'" . implode("','", array_map('dbesc', explode(",", str_replace(" ", "", $a->config['admin_email'])))) . "'";
```

Comment:
Code standards: Although I personally would rather we use single quotes everywhere, until we actually do please use the most common quoting style in the file/function/line.